react-function-components/src/index.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const customButton = <button>Click Me!</button>;
```

Comment:
`CustomButton` should be a function that returns the JSX of what you want to display:
```JSX
function CustomButton() {
  return <button>Click Me!</button>;
}
```